### DISCUSSION TITLE
I can not install lastest version of OsmAnd~ from F-droid (3.9.10)

### DISCUSSION THREAD

**Discussion by Dani3I** [osmandapp/OsmAnd#12341] [General]
### Description
I tried to install the latest version (3.9.10) on my phone from the f-droid store and an error pops up that the app was unable to install due to an unknown error. I tried restarting the phone but it didn't help.
So I tried to install the previous version (3.9.8) and the installation was successful.

### How to reproduce?
Click on "install" button in F-droid store and wait for error message.

### Your Environment
OsmAnd Version: "OsmAnd~" version 3.9.10 from F-droid
Android/iOS version: Android 11 (MIUI 12.5.1)
Device model: POCO X3 Pro

**Maps used (online or offline):**   
N/A

**Comment by Dani3I** [comment]
Either I don't know where to check it or my issue was closed for no reason. Why has this happened?

**Reply by vshcherb** [reply]
OsmAnd team doesn't build OsmAnd~ for F-Droid

**Reply by Dani3I** [reply]
@vshcherb
It seems a bit odd to me that you assumed it was a problem due to who built it, especially since I suspect the source code is provided by the OsmAnd team, but whatever.

Anyway, it turned out to be due to PlayProtect, but actually due to the fact that I have a password set for certain apps that I have to type in before launching them (MIUI gives this option), and among them is also play store:
When the app (in version 3.9.10, because in version 3.9.8 this does not occur) was installing, at the end of the installation a warning from PlayProtect pops up and was saing something like this: "Blocked by PlayProtect." "This application may collect data that can be used to track you.".

But in my case, the system first tried to ask me for my password, which for some reason interrupted the installation, which in turn immediately disabled the password prompt as well.
I discovered this fact when, during normal use of the phone, I first unlocked the play store with a password and after that tried to install the app from f-droid (password queries reset after each lock of the phone).

**Reply by Dani3I** [reply]
In fact it is strange that PlayProtect blocks this application in the F-Droid store but not in the play store and that this happens with the latest version but not with older

**Reply by Dani3I** [reply]
Now I read a bit more about the f-droid version and by now I realized that it probably wasn't even the osmand team that came up with the idea to release the F-Droid version.

I thought that it was officially the people from OsmAnd team who wanted the application to be on F-Droid, but for example someone from F-Droid is simply responsible only for building the application. This is a bit confusing, because probably all links from F-Droid lead to official OsmAnd pages.

**Reply by vshcherb** [reply]
This is an old question, Google starts blocking all apps that don't use Google Play Services. F-Droid doens't track anything and doesn't use Google Play Services, though Google tries to prove other way around.

**Reply by mariush444** [reply]
Can disable "Google Play Protect" solve the problem?

**Reply by Dani3I** [reply]
> 
> 
> Can disable "Google Play Protect" solve the problem?

Oh I didn't write it, but the problem I described was the only problem and after "entering the password to unlock the Play Store application" everything worked without any major problems.
Play Protect gave me the option to choose whether I want the application to remain installed anyway, so I do not have to turn off Play Protect.
